db-engine/src/main/java/com/splicemachine/db/iapi/sql/dictionary/DataDictionary.java

```diff
@@ -68,7 +68,7 @@
     /** The conglomerate id of the first user table */
     // NOTE: JC - this constant is also defined in (splice) EnvUtils. When adding a new sys table, this
     // number will need to be increased in BOTH places.
-    long FIRST_USER_TABLE_NUMBER = 1458;
+    long FIRST_USER_TABLE_NUMBER = 1568;
```

Comment:
This should be increased earlier. For anyone who introduces new system tables, bump up this value.